app/src/main/java/org/mozilla/fenix/settings/DataChoicesFragment.kt

```diff
@@ -40,16 +43,14 @@ class DataChoicesFragment : PreferenceFragmentCompat() {
                 } else {
                     context.components.analytics.metrics.stop(MetricServiceType.Marketing)
                 }
-            } else if (key == getPreferenceKey(R.string.pref_key_experimentation)) {
-                val enabled = context.settings().isExperimentationEnabled
-                context.components.analytics.experiments.globalUserParticipation = enabled
             }
         }
     }
 
     override fun onResume() {
         super.onResume()
         showToolbar(getString(R.string.preferences_data_collection))
+        updateStudiesSection()
```

Comment:
Why are we only doing this `onResume`?

Reply:
Because as now the study section can be changed from a sub screen and we need to keep the label **ON/OFF** in sync.

Imagine you are in the **"Data Collection Screen"** screen, and the **Studies** sections says ON, then you go to the Studies screen and switch the preferences to **OFF** the label on in the **"Data Collection Screen"** screen has to be updated.

![image](https://user-images.githubusercontent.com/773158/128904887-55740c8c-f78e-4035-9c4d-243c0e91a184.png)

Reply:
:)